clang/lib/Sema/SemaInit.cpp

```diff
@@ -465,7 +460,8 @@ class InitListChecker {
   void FillInEmptyInitForField(unsigned Init, FieldDecl *Field,
                                const InitializedEntity &ParentEntity,
                                InitListExpr *ILE, bool &RequiresSecondPass,
-                               bool FillWithNoInit = false);
+                               bool FillWithNoInit = false,
+                               bool MaybeEmitMFIWarning = true);
```

Comment:
This is not an initialism that we can expect future readers of the code to understand. If you want to keep this short, how about something like `WarnIfMissing`?

Reply:
Okay, I changed the name.